### ISSUE TITLE
Assets Dumper repository not available to the public

### ISSUE THREAD

**Issue by angeloanan** [PretendoNetwork/mario_kart_8_website#4]
The README mentions the requirement to run the MK8 UI assets dumper as a prerequisite to fill in missing assets / images from the site. The said repository is not available to the public.

As an "outsider" trying to contribute towards the project, is there any other way to get this dump?
If there is, please consider changing the README to reflect it.

**Comment by EpicUsername12** [comment]
Yes, the asset dumper is private, and I don't have the perms to make it public (cc @jonbarrow )

I just made it available here too (warning, the code is absolutely ugly and in 2 languages): https://github.com/EpicUsername12/mk8_ui_assets_extractor

**Comment by jonbarrow** [comment]
I can definitely make that public here, is that still something that needs to be done now that it's on your own GH though?
